CHANGELOG.md

```diff
@@ -1,11 +1,14 @@
 # CHANGELOG
 
 ## Next Release
+### Internal
```

Comment:
This should be `Feature` right?

Reply:
It should be in "Changes", I think. A feature is something that the user can use, but this is not something that the nurse can take action on.

Reply:
Can you tell me the difference between Internal | Changes ?

Reply:
This was my bad, I should have caught this when we paired on this PR.

Internal -> "Not a user facing change"
"Feature" -> "A new tool which the user can use"
"Changes" -> "A user facing change, but which is not in control of the user"
"Fixes" -> "Bug fixes"